oeps/processes/oep-0055-project-maintainers.rst

```diff
@@ -0,0 +1,309 @@
+responsible for the overall oversight and stewardship of the Open edX
+project. The TOC itself was established and is governed by a formal
+charter.  You can read the full document `here
+<https://openedx.org/wp-content/uploads/2021/12/TCRIL-Technical-Oversight-Committee-Charter-rev.11-16.21.pdf>`_.
+
+Context
+-------
+Historically, the Open edX project has not had a well-defined role
+of maintainer.  At times, community members stepped into this role
+informally.  The role was also part of a broader "owner" role at edX.
+Owners were also responsible for the health of the component they
+owned both in the repository and in production; owners answered the
+pager when something went wrong.
+
+The role of the repository or project maintainer is both a community
+focused and a technical role[#]_.  Maintainers will be the public face
+of each component and should be welcoming and available.
+
+Non-technical responsibilities of maintainers include:
+
+* Effective communication with community members, other maintainers,
+  and project leadership
+* Disciplined management to ensure that maintenance work is done
+* Enforcement of the project's standards
+* The ability to solve or escalate issues facing users of the
+  component
+
+Technical responsibilities of maintainers include:
+
+* Understanding the current component architecture and implementation
+* Having a technical vision for the evolution of the component
+* The ability to assess the appropriateness of proposed changes to or
+  extensions of the component
+* The role of the component in the overall platform and how it
+  interacts with other components
+
+It is acceptable for a component to be maintained by a team or group
+as long as that arrangement does not reduce the reachablility of the
+maintainer or reduce accountability.
+
+Prerequisites
+-------------
+In order to be a repository maintainer, you must
+
+* Be an active member of the Open edX community who has contributed to
+  the project in the last 6 months.  The types of contributions that
+  are relevant will match those defined in the `core contributor
+  program`_
+* Abide by the `Open edX Code of Conduct
+  <https://openedx.org/code-of-conduct/>`_
+* Be a core contributor for the repository in question who has
+  demonstrated excellence in the role of core contributor.  From
+  time-to-time, in the case of a strong candidate and project need,
+  core contributor status and maintainer status may be conferred at
+  the same time.
+
+Bootstrapping
+-------------
+In order to ensure an orderly roll-out of the formal maintainer role,
+existing 2U staff in the owner role will transition into the role of
+maintainer for the components that they own.
+
+Community members who are acting in informal maintainer roles, will
+also be asked to become named maintainers.
+
+A list of all components without maintainers will be created and
+published.
+
+An asynchronous maintainer scrum-of-scrums will be created as a Slack
+channel.
```

Comment:
[suggestion] I'm not sure you want to be this prescriptive in an OEP, as if you pivot what this slack channel means or move to a different solution you may not remember to update the OEP.

Reply:
Fair point.  Thoughts on how to ensure that the SOSs self-organizes?

Reply:
For the CC program, we have a Project manager (also a core contributor), to make sure everyone is in sync and to help resolve blockers. That PM is currently trialing the idea of loose sprints, and collecting statuses in an app called Friday. So practically speaking, you make a Project Management CC role to ensure that the maintainer program runs properly.

In OEP-54 I have the notion of "program administrators" who are tCRIL employees who can be impartial/neutral in the nomination process, and who basically make sure everything is working right (they have a few responsibilities enumerated in the document, but basically, an "escalate to this body of people if things are going wrong"). I could see a similar structure - or even the same set of program administrators from OEP-54 - who are there as a backstop to the overall program.

Also on re-reading this I'm not sure what it means to call the group of maintainers a "scrum of scrums", unless I'm supposed to be implicitly inferring that maintainers are also supposed to act like scrum masters of some team?

Reply:
I don't think I answered your question though. What about `Asynchronous communication channels will be in effect for maintainers to connect with one another; initially, this will be a Slack channel.`

This lays out the goal first, with a proposed implementation second, and if we pivot the sentence isn't really out of date.